stage_descriptions/auth-12-ya4.md

```diff
@@ -0,0 +1,108 @@
+In this stage, you'll add support for implementing the `default` user.
```

Comment:
This is a bit of a weird stage - the user has already added support for default in many places before this. In practice we're just testing whether `nopass` can be removed from `default` and if that's honored. Let's remove this, don't think it's worth covering